packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -88,8 +96,14 @@ export interface SdkEmitterOptions {
   "flatten-union-as-enum"?: boolean;
   "api-version"?: string;
   "examples-dir"?: string;
-  "emitter-name"?: string;
   namespace?: string;
+  license?: {
```

Comment:
Should there be a displayString property that gives the default display?

Reply:
if there is not `license` config, emitter should not generate any license header or license file.

Reply:
oh yes. emitter could use `sdkPackage.license` info. if it is not undefined, emitter could use `sdkPackage.license.header` for the license header comment and use `sdkPackage.license.description` for the separated license file content directly.

Reply:
Understood, but would it make sense to have an API for emitters to use? e.g.
```function getLicenseDisplayString(license: LicenseInfo) : string```

Reply:
for current design, all the config for license should be transparent for emitter, i.e. emitter only needs to use `context.sdkPackage.license` to get the display string, no need to care about the raw config in `tspconfig.yaml`.

Reply:
i'll merge this pr to catch the coming release. we could discuss further if more helper function is needed.